### PR TITLE
fix(auth): replace user id with nanoId

### PR DESCRIPTION
**Pull request by Abingcbc**

### CODE REVIEW

File: pkg/auth/utils/config.go [hunk 1]
```diff
@@ -46,32 +51,38 @@ func CreateOrUpdateKubeConfig(uid string) error {
...
 	})
-	user, err := resource.Get(context.Background(), uid, metav1.GetOptions{})
-	if k8sErrors.IsNotFound(err) {
-		_, err := resource.Create(context.TODO(), &unstructured.Unstructured{Object: map[string]interface{}{
+	labelSelector, _ := labels.Parse("uid=" + uid)
+	users, err := resource.List(context.Background(), metav1.ListOptions{LabelSelector: labelSelector.String()})
+	if err != nil {
+		return err
+	}
+	if len(users.Items) == 0 {
```
Comment (zzjin) [inline pkg/auth/utils/config.go:59]: Should we check len(users.Items) must be 1?

File: pkg/auth/utils/config.go [hunk 2]
```diff
@@ -46,32 +51,38 @@ func CreateOrUpdateKubeConfig(uid string) error {
...
-	} else if err != nil {
-		return err
 	} else {
 		// To trigger controller reconcile
-		user.Object["metadata"].(map[string]interface{})["labels"].(map[string]interface{})["updateTime"] = time.Now().Format("T2006-01-02T15-04-05")
-		_, err = resource.Update(context.Background(), user, metav1.UpdateOptions{})
-		if err != nil {
-			return err
+		for _, user := range users.Items {
+			user.Object["metadata"].(map[string]interface{})["labels"].(map[string]interface{})["updateTime"] = time.Now().Format("T2006-01-02T15-04-05")
```
Comment (cuisongliu) [inline pkg/auth/utils/config.go:81]: The writing is a bit complicated
```
labels:=user.GetLabels()[updateTime]=xxx
user.SetLabels(label)
```

**Review by zzjin — CHANGES_REQUESTED** [review]
Need to add old users sync script.

**Comment by fanux** [comment]
@zzjin I accept to clean up the old users directly, the migration cost is too high, and such things will not happen in the future

**Review by zzjin — APPROVED** [review]

**Review by zzjin — APPROVED** [review]

**Pull request merged**